x/track.rb

```diff
@@ -8,6 +10,12 @@ def self.all
       end
     end
 
+    def self.active
```

Comment:
I'd rather not have this method here. The `X::Track` class feels like it should be as generic as possible.